src/db_iterator.rs

```diff
@@ -33,7 +33,11 @@ pub type DBRawIterator<'a> = DBRawIteratorWithThreadMode<'a, DB>;
 /// ```
 /// use rocksdb::{DB, Options};
 ///
-/// let path = "_path_for_rocksdb_storage4";
+/// let tempdir = tempfile::Builder::new()
+///     .prefix("_path_for_rocksdb_storage4")
+///     .tempdir()
+///     .expect("Failed to create temporary path for the _path_for_rocksdb_storage4.");
+/// let path = tempdir.path();
```

Comment:
I'm not sure about that myself, but perhaps it would be better to reduce clutter in the examples by hiding the creation of the temporary directory. This can be done by using the `#` prefix. The disadvantage of such approach is that the temp variable will be implicit for a user of the documentation.

Reply:
I would say we should leave the examples/code comments as is (not using a temp path/directory) since that's what users will do in production. The temp directory is good for tests to avoid issues with building/testing the library.

Reply:
Yes. I didn't want to have to make these changes, but once I added the read-only CI test (to detect future regressions) then I noticed that the doc tests had the same problem.

Reply:
ok, in that case, I guess what you have here is good enough.